designs/NuGet-RemoteFileService.md

```diff
@@ -0,0 +1,70 @@
+
+# Spec Name
+
+* Status: In Review
+* Author(s): [Rob Relyea](https://github.com/rrelyea)
+* Issue: [10151](https://github.com/NuGet/Home/issues/10151) INuGetRemoteFileService - Fetch Images and embedded licenses for codespaces and normal execution
+
+## Problem Background
+
+With Codespaces support added to NuGet client, our code needs to be able to handle all packages and communication with feeds happening on a server node. This is architected as async method calls to services that are remoted. In most cases, we also have chosen to have 1 code path, for normal execution and remoted execution. These services are still used in the normal execution, however, the service runs in the same process, as opposed to being remoted across the network.

```

Comment:
Referring to standalone and Codespaces-connected clients as "normal" and "remoted" confusing, primarily over the use of the word "normal."

Reply:
switched to standalone and Codespaces-connected.  see next iteration.